src/window/intro_video.c

```diff
@@ -33,8 +48,12 @@ static void draw_foreground(void)
 static void handle_mouse(const mouse *m)
 {
     if (m->left.went_up || m->right.went_up || video_is_finished()) {
+        sound_music_stop();
```

Comment:
There should be no need to stop music here, since the video player manages its own video.

Reply:
It is, otherwise you can hear ROME1 start playing between the videos :/